package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "perseus",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:Khan/perseus.git",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "config/build"
+  ],
+  "engines": {
+    "node": ">=12"
```

Comment:
Given this, I wonder if we should have a github action that runs all of our tests in a Node 12 env even if we don't build things in that env.

Reply:
webapp hasn't been fully updated to use node 16 and I'm not sure what versions of node the mobile repo works with.